flutter/lib/ui/result_screen.dart

```diff
@@ -75,11 +75,11 @@ class _ResultScreenState extends State<ResultScreen>
         benchmarkResult = benchmark.performanceModeResult;
         final throughput = benchmarkResult?.throughput;
         textResult = throughput?.toStringAsFixed(2) ?? 'N/A';
-        numericResult = (throughput ?? 0) / benchmark.info.maxThroughput;
+        numericResult = (throughput ?? 0.0) / benchmark.info.maxThroughput;
       } else if (_screenMode == _ScreenMode.accuracy) {
         benchmarkResult = benchmark.accuracyModeResult;
-        textResult = benchmarkResult?.accuracy ?? 'N/A';
-        numericResult = benchmarkResult?.numericAccuracy ?? 0.0;
+        textResult = benchmarkResult?.accuracy?.string ?? 'N/A';
+        numericResult = (benchmarkResult?.accuracy?.value ?? 0.0) / 100.0;
```

Comment:
If we normalize the accuracy between `0.0` and `1.0` then there is no need to have
`virtual float ComputeNormalizedAccuracy() { return ComputeAccuracy() * 100; }`
and then `value / 100.0` here

Reply:
Done